lib/drives/pages/create_drive_page.dart

```diff
@@ -170,10 +171,9 @@ class _CreateDriveFormState extends State<CreateDriveForm> {
         );
       } on AuthException {
         //todo: change error message when login is implemented
-        ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
looking over the code i'm not sure if we even need this part. What could be an Exception here? I mean that all values are valid is checked by the validators and. If I have the rights to create a drive is checked by the RLS and that should not return an Exception.

Reply:
Do you think so? Now I'm not sure, what is returned on `insert` when RLS prevents it?

But that should be done in another PR anyways.